brightlayer-ui/react-component-library
# 3060 toolbar menu mui 5 styled

Fixes # .


#### Changes proposed in this Pull Request:
- Update to use Mui-v5 styled components
- Update component to extend sx prop
- Update component docs for ToolbarMenu

**Also would like a suggestion on an example code snippet for this in the docs


#### Screenshots:
Temporary ToolbarMenu
![image](https://user-images.githubusercontent.com/37150565/165996403-70e5f47a-9ab6-49c3-8de3-b269d47cae99.png)



#### To Test:
- There is a showcase branch [feature/3050-channel-value-mui-v5-styles-update](https://github.com/brightlayer-ui/react-showcase-demo/tree/feature/3050-channel-value-mui-v5-styles-update) that this can be tested in the ```SharedAppBar```

```
                 <ToolbarMenu
                    label={'Toolbar-menu'}
                    menuGroups={menuGroups}
                    // style={{ backgroundColor: '#ffbb00' }} // specificity 1
                    classes={{ rotatedDropdownArrow: classes.green }} // specificity 2
                    // className={classes.blue} // specificity 3
                    sx={{ backgroundColor: '#bb00ff' }} // specificity 4
                ></ToolbarMenu>
```

Ensure that there are no breaking changes and that the old API properties still work the same and that you can still style inline styles using our classes prop.
Test sx prop extension.
